utoipa-swagger-ui/build.rs

```diff
@@ -105,7 +105,8 @@ fn replace_default_url_with_config(target_dir: &str) {
     .iter()
     .collect::<PathBuf>();
 
-    let swagger_initializer = fs::read_to_string(&path).unwrap();
+    let mut swagger_initializer = fs::read_to_string(&path).unwrap();
+    swagger_initializer = swagger_initializer.replace("layout: \"StandaloneLayout\"", "nothing: false");
```

Comment:
Maybe just directly replace it to "Nothing" instead :)
```suggestion
    swagger_initializer = swagger_initializer.replace("layout: \"StandaloneLayout\"", "");
```